Pre go-live cleanup of ff docs (#215)
harness/developer-hub: docs/feature-flags/4-ff-sdks/1-sdk-overview/1-client-side-and-server-side-sdks.md

|  |  |  |
| --- | --- | --- |
| **Parameter** | **Client Side SDK** | **Server Side SDK** |

> | **Users** | Designed to be used in applications, your users run directly on their own devices, such as mobile, desktop, and web applications. Optimized to be used by a single user and low bandwidth consumption. Examples: JavaScript, iOS, Android, React Native SDKs.  | Designed to be used in server-side applications such as web servers and backend services. Optimized to be used in multi-user and secure environments.Examples, Java, Go, and Python SDKs. |

Reviewer: ```suggestion
| **Users** | Designed to be used in applications, your users run directly on their own devices, such as mobile, desktop, and web applications. Optimized to be used by a single user and provide low bandwidth consumption. Examples: JavaScript, iOS, Android, React Native SDKs.  | Designed to be used in server-side applications, such as web servers and backend services. Optimized to be used in multi-user and secure environments. Examples, Java, Go, and, Python SDKs. |
```

> | **Security** | The SDKs are embedded in your applications and run on the browser or on mobile devices. Because of which these SDKs can be compromised by users when unpacking a mobile app or using the browser's developer tools to inspect the page. You should not use a server-side SDK key in client-side applications. Client-side SDKs only perform an evaluation of a flag and receive the result.  They don't store all data about a flag. | The SDKs are embedded in applications that run on your servers such as web servers or backend servers. These environments are comparatively safe. Server-side SDKs download all the feature flags that you have defined in a project and store them in memory. |

Reviewer: ```suggestion
| **Security** | The SDKs are embedded in your applications and run on the browser or on mobile devices. Therefore, these SDKs can be compromised by users when unpacking a mobile app or using the browser's developer tools to inspect the page. You should not use a server-side SDK key in client-side applications. Client-side SDKs only perform an evaluation of a flag and receive the result. They don't store all data about a flag. | The SDKs are embedded in applications that run on your servers, such as web servers or backend servers. These environments are comparatively safe. Server-side SDKs download all the feature flags that you have defined in a project and store them in memory. |
```
